docs/api/telegram.rst

```diff
@@ -360,6 +360,41 @@ about its business.
 
       .. versionadded:: 0.6
 
+   .. py:method:: send_gif([path=None, file_id=None, duration=None, width=None, height=None, caption=None, thumb=None, reply_to=None, attach=None, extra=None, notify=True])
+
+      Send an animation to the user. You can specify the animation by passing its *path*,
+      its *url*, or its Telegram *file_id*. Only one of these arguments must be passed.
+
+      You may optionally specify the *duration*, the *width* and the *height* of the GIF.
+      If the GIF track you're sending is in reply to another message,
+      set *reply_to* to the ID of the other :py:class:`~botogram.Message`.
+
+      The *attach* parameter allows you to attach extra things like
+      :ref:`buttons <buttons>` to the message.
+
+      The *notify* parameter is for defining if your message should trigger
```

Comment:
It's better to say "is to define" than "is for defining"

Reply:
This will be fixed in another PR, is a global problem